src/Microsoft.AspNetCore.ResponseCaching/Internal/DistributedResponseCacheStore.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.AspNetCore.ResponseCaching.Internal
 {
-    internal class DistributedResponseCache : IResponseCache
+    internal class DistributedResponseCache : IResponseCacheStore
```

Comment:
DistributedResponseCacheStore.

Why is this internal? recommend public in the Internal namespace.